src/asset.ts

```diff
@@ -308,4 +308,14 @@ export default class Asset extends AinftBase {
     const trailingUrl = `credit/${symbol}/lockup`;
     return this.sendRequest(HttpMethod.POST, trailingUrl, body);
   }
+  /**
```

Comment:
indentation? + could you add an empty line above?

Reply:
Added, thanks!